python-flask/python-flask-example.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, request, abort
+import base64
+import json
+import pprint
+import datetime
+app = Flask(__name__)
+
+@app.route('/', defaults={'path': ''}, methods=['POST'])
+@app.route('/<path:path>', methods=['POST'])
+def log_payload(path):
+    # Get request payload
+    if request.content_type != 'application/json':
+        # Unsupported Media Type
+        return 'Content type is not JSON', 415
+
+    # Ensure that payload is a dictionary
+    payload = request.json
+    if not isinstance(payload, dict):
+        # Payload not JSON, server error
+        return 'Payload is not JSON', 500
+
+    # Get message data
+    timestamp = datetime.datetime.now()
+    message = payload.get('message',{})
+    if message:
+        # Get device ID
+        id = message.get('attributes',{}).get('deviceId','')
+        # Get base64 encoded data payload
+        data = base64.b64decode(message.get('data',''))
+        print('----- Got message from {} ({}) -------'.format(id, timestamp))
+        pprint.pprint(json.loads(data))
+    else:
+        print('----- Got message that could not be parsed ({}) -----'.format(timestamp))
+        print(payload)
+    return 'OK', 200
```

Comment:
Could use the `http` module to avoid magic values

```python
>>> from http import HTTPStatus
>>> HTTPStatus.OK
<HTTPStatus.OK: 200>
>>> HTTPStatus.OK.phrase
'OK'
>>> HTTPStatus.OK.value
200
```

but it's just a minor thing in the case of this example, IMO.

Reply:
Flask did not support just sending e.g `return HTTPStatus.OK` so will leave example as it is to leave out complexity..